pkgs/development/python-modules/nexusformat/default.nix

```
@@ -0,0 +1,57 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+
+  # build-system
+  setuptools,
+  setuptools-scm,
+  wheel,
+
+  # tests
+  pytest,
+
+  # dependencies
+  h5py,
+  hdf5plugin,
+  numpy,
+  scipy,
+}:
+
+buildPythonPackage rec {
+  pname = "nexusformat";
+  version = "1.0.6";
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-UxU3PA/2r/uamdysbfC0L2JinHgfkXhssHIo2hf3zlA=";
+  };
+
+  pyproject = true;
+
+  build-system = [
+    setuptools
+    setuptools-scm
+    wheel
+  ];
+
+  dependencies = [
+    h5py
+    hdf5plugin
+    numpy
+    scipy
+  ];
+
+  nativeCheckInputs = [ pytest ];
+
+  checkPhase = ''
+    pytest
+  '';
+
+  meta = with lib; {
+    description = "Python API to open, create, and manipulate NeXus data written in the HDF5 format.";
```

```suggestion
    description = "Python API to open, create, and manipulate NeXus data written in the HDF5 format";
```
Please remove the period.
https://nixos.org/manual/nixpkgs/unstable/#var-meta-description